src/components/QuickGrid/QuickGrid.scss

```diff
@@ -1,352 +1,362 @@
+    }
+}
+
+.empty-header-column {
+    border-right: 0px;
+}
+
+.grid-header-column:last-child,
+.ReactVirtualized__Table__rowColumn:last-child {
+    border-right: 0;
+}
+
+.ReactVirtualized__Table__rowColumn:first-child {
+    padding-left: 22px;
+}
+
+.ReactVirtualized__Table__headerRow {
+    font-weight: 700;
+    text-transform: uppercase;
+    display: -webkit-flex;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-direction: row;
+    -moz-box-orient: horizontal;
+    -moz-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -webkit-align-items: center;
+    -moz-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
+
+.ReactVirtualized__Table__row {
+    display: -webkit-flex;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-flex-direction: row;
+    -moz-box-orient: horizontal;
+    -moz-box-direction: normal;
+    -ms-flex-direction: row;
+    flex-direction: row;
+    -webkit-align-items: center;
+    -moz-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
+
+.ReactVirtualized__Table__headerTruncatedText {
+    display: inline-block;
+    max-width: 100%;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+}
+
+.ReactVirtualized__Table__headerColumn,
+.ReactVirtualized__Table__rowColumn {
+    min-width: 0px;
+    font-size: 13px;
+    color: $emphasized-text-color;
+}
+
+.ReactVirtualized__Table__rowColumn {
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    border-right: 1px solid $search-border-color;
+    height: 100%;
+    padding-left: 10px;
+    box-sizing: border-box;
+    padding-top: 6px;
+}
+
+.grid-component-cell {
+    .status-critical {
+        color: $red-color;
+    }
+    .status-warning {
+        color: $yellow-color;
+    }
+    .status-ok {
+        color: $emphasized-text-color;
+    }
+}
+
+.ReactVirtualized__Table__sortableHeaderColumn {
+    cursor: pointer;
+}
+
+.ReactVirtualized__Table__sortableHeaderIconContainer {
+    display: -webkit-flex;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-align-items: center;
+    -moz-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+}
+
+.ReactVirtualized__Table__sortableHeaderIcon {
+    -webkit-flex: 0 0 24px;
+    -moz-box-flex: 0;
+    -ms-flex: 0 0 24px;
+    flex: 0 0 24px;
+    width: 20px;
+    height: 20px;
+    fill: currentColor;
+    margin-bottom: 3px;
+}
+
+.border-column-cell {
+    border-right: 1px solid #ddd;
+    box-sizing: border-box;
+}
+
+//DropToolbar
+.group-drop-toolbar {
+    width: calc(100% - 5px);
+    height: 38px;
+    padding: 5px 2px 2px;
+    margin-bottom: 15px;
+    float: left;
+    border: 1px solid #c8c8c8;
+    .group-drop-empty {
+        margin: 7px auto;
+        width: 200px;
+        display: block;
+        color: #4b4949;
+    }
+    > .expand-collapse-buttons {
```

Comment:
This is the only real change in the file. Each line had a single space padding, will be removed now.